src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,38 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import './TodoInfo.scss';
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/types';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    title,
+    completed,
+    user,
+    id,
+  } = todo;
+
+  return (
+    <article
+      data-id={id}
+      className={classNames(
+        'TodoInfo',
+        {
+          'TodoInfo--completed': completed,
+        },
+      )}
+    >
+      <h2 className="TodoInfo__title">
+        {title}
+      </h2>
+      {user
+        && (<UserInfo user={user} />
+        || <div>No user found</div>)}
```

Comment:
It's better to use ternary operator instead of `&&` and `||` for conditional rendering. Replace with: `user ? <UserInfo user={user} /> : <div>No user found</div>`